tasks/parse-pdr/index.js

```diff
@@ -198,18 +198,19 @@ const convertFileGroupToGranule = async ({
     collectionVersion: version,
   });
 
-  let provider;
+  let providerName;
   if (fileGroup.get('NODE_NAME')) {
     const host = fileGroup.get('NODE_NAME').value;
 
-    provider = await getProviderByHost({ prefix, host });
+    const provider = await getProviderByHost({ prefix, host });
+    providerName = provider.id;
   }
 
   return {
     dataType,
     version,
     files,
-    provider,
+    provider: providerName,
```

Comment:
I haven't looked deeply into this at all - but is there something in the PDR workflow tasks code that is expecting `granule.provider` to be an object? Does this even need to be there at all?

Does `sync-granule` look at/respect `granule.provider` if it exists?

Reply:
>Does sync-granule look at/respect granule.provider if it exists? 
I don't believe so (on either count) , no,it looks like the granule fetcher doesn't use it, and interestingly: 


    // return all files, the renamed files don't have the same properties
    // (name, size, checksum) as input file
    log.debug(`returning ${JSON.stringify(stagedFile)}`);
    return [stagedFile].concat(versionedFiles.map((f) => (
      {
        bucket: destinationBucket,
        name: path.basename(f.Key),
        path: file.path,
        filename: S3.buildS3Uri(f.Bucket, f.Key),
        size: f.size,
        fileStagingDir: stagingPath,
        url_path: this.getUrlPath(file),
      })));
      
 
strips that value out, and that matches what you see in practice in this test run.    See: ticket re: making expected inputs/outputs/schema consistent.     That's also consistent with the input/output schema of the task.

Reply:
>    Does sync-granule look at/respect granule.provider if it exists?

No.  It refers to 

```
exports.syncGranule = function syncGranule(event) {
  const config = event.config;
  const input = event.input;
  const stack = config.stack;
  const buckets = config.buckets;
  const provider = config.provider;
```

and that's part of most workflow configurations as 

```
          "task_config": {
            "buckets": "{$.meta.buckets}",
            "provider": "{$.meta.provider}",
```

Reply:
I had a prior comment re granule.files.provider, and confirmed *that* also isn't referenced, fwiw.

Reply:
Queue-granules does expect this, as it uses it to populate the `meta.provider` field for the sync-granule workflow. I don't think we have a choice but to keep it.